tracer/tracer.go

```diff
@@ -554,23 +554,22 @@ func loadAllMaps(coll *cebpf.CollectionSpec, cfg *Config,
 	// On modern systems /proc/sys/kernel/pid_max defaults to 4194304.
 	// Try to fit this PID space scaled down with cfg.OffCPUThreshold into
 	// this map.
-	adaption["sched_times"] = (4194304 / support.OffCPUThresholdMax) * cfg.OffCPUThreshold
+	adaption["sched_times"] = (4194304 * cfg.OffCPUThreshold) / support.OffCPUThresholdMax
```

Comment:
Isn't this sizing excessive if we bound it by number of possible PIDs? For example, if we set `off-cpu-threshold` to `1000`, we'll end up with a map that takes up roughly `4194304*16 = 64MB` since we pre-allocate. 

Generated off-cpu events are bounded by cores and scheduler call frequency, so for example we could size this map as: `next_pow2(cores*1000*pollfreq)` using 1000Hz as an estimate of scheduler call frequency which should consume far less memory.

Reply:
Unrelated to this PR, created #320 to track this.